From ff3f75ab44bbace07785441452ff3a4cc8fdce79 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:26 -0400
Subject: [tools] perf/mem: Remove unused parameter from dump_raw_samples()

Message-id: <1408953210-26343-199-git-send-email-jolsa@redhat.com>
Patchwork-id: 88961
O-Subject: [PATCH RHEL7.1 BZ1133083 198/322] perf mem: Remove unused parameter from dump_raw_samples()
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 8b640cc4c56cee14bfe5cfb4dbb372ac66d5ec6b
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Thu Dec 19 17:00:45 2013 -0300

description
===========
The 'evsel' parameter is not used, ditch it, reducing the function
signature.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-mem.c b/tools/perf/builtin-mem.c
index 31c00f1..2e3ade6 100644
--- a/tools/perf/builtin-mem.c
+++ b/tools/perf/builtin-mem.c
@@ -62,7 +62,6 @@ static int
 dump_raw_samples(struct perf_tool *tool,
    union perf_event *event,
    struct perf_sample *sample,
-   struct perf_evsel *evsel __maybe_unused,
    struct machine *machine)
 {
  struct perf_mem *mem = container_of(tool, struct perf_mem, tool);
@@ -112,10 +111,10 @@ dump_raw_samples(struct perf_tool *tool,
 static int process_sample_event(struct perf_tool *tool,
     union perf_event *event,
     struct perf_sample *sample,
-    struct perf_evsel *evsel,
+    struct perf_evsel *evsel __maybe_unused,
     struct machine *machine)
 {
- return dump_raw_samples(tool, event, sample, evsel, machine);
+ return dump_raw_samples(tool, event, sample, machine);
 }
 
 static int report_raw_events(struct perf_mem *mem)
-- 
1.7.1